s3/replication/common/tests/system/test_head_object.py

```diff
@@ -50,10 +50,16 @@ async def main():
     bucket_name = config.source_bucket_name
     request_id = "dummy-request-id"
 
+    # For Cortx, Default value of version id=None
+    # For AWS, please provide the version id
     head_obj = S3AsyncHeadObject(session, request_id,
-                                 bucket_name,
-                                 object_name)
-    await head_obj.get()
+                                 bucket_name, object_name,
+                                 None)
+
+    # Pass the part_number to get specific part's information
```

Comment:
can be removed.

pass version_id here, rather than passing None and adding comment.

Reply:
done